README.md

```diff
@@ -37,46 +41,45 @@ endpoint permissions in the documentation as well you need to name the function
+| **generateUI**							                     | boolean		     | 	✅	| Whether [Swagger UI](https://swagger.io/tools/swagger-ui/) should be generated.                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| **middlewares**							                    | object		      | 	❌	| Configuration parameters for parsing middlewares.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| **middlewares**.formatter					            | function		    | 	❌	| Custom formatter function for middleware. If not used, the default one will be used.                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| **middlewares**.middlewareName			         | string		      | 	✅	| Name of the middleware responsible for handling API permissions.                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| **middlewares**.closure					              | string		      | 	✅	| Name of the permission middleware closure.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
@kot-1999 rename `Name of the permission middleware closure.` to `Name of the middleware closure.`